app/src/util/tools.ts

```diff
@@ -121,6 +124,7 @@ export const computePriceAfterTrade = (
 
 export const isAddress = (address: string): boolean => {
   try {
+    logger.log(`Address '${address}' doesn't exist`)
```

Comment:
Shouldn't this be in the catch block?

Reply:
Ohh yes, sorry about that, I will move it, thanks